README.md

```diff
@@ -21,3 +21,24 @@ And include extension.neon in your project's PHPStan config
 includes:
   - vendor/timeweb/phpstan-enum/extension.neon
 ```
+
+## Install for Local Development 
+
+### With docker:
+
+```
+git clone git@github.com:timeweb/phpstan-enum.git
+cd phpstan-enum
+make docker-build
+make install
+make phpunit
+```
+
+### Without docker (localy installed actual version of php, composer, etc):
```

Comment:
I think instructions about working without Docker are not correct.
Scripts in `tools` will complain about non-existent `docker` command or
image.  Am I missing something?

Reply:
Fixed.

Reply:
Aha, one `$PATH` to rule them all :)

What if there is no `phpunit` command in `$PATH`.  There is always one in
`vendor/bin`.  But if you put this directory before `tools`, `tools/phpunit`
will never be called.

What do you think about single script solution I mentioned erlier?